renovate.json

```diff
@@ -1,38 +1,6 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "platform": "github",
-  "labels": ["renovate"],
-  "extends": [
-    "config:base",
-    ":rebaseStalePrs",
-    "docker:disableMajor"
-  ],
-  "enabledManagers": [
-    "nuget",
-    "dockerfile",
-    "github-actions"
-  ],
-  "stabilityDays": 3,
-  "prHourlyLimit": 0,
-  "prConcurrentLimit": 0,
-  "branchConcurrentLimit": 0,
-  "dependencyDashboard": false,
-  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
-  "nuget": {
-    "extends": [":disableMajorUpdates"]
```

Comment:
We will use shared config that split major than the rest:
- Leave options to maintainers
- At one point major version reach End of life and need to be uddated